.ci/jenkins/dsl/jobs.groovy

```diff
@@ -118,14 +156,14 @@ void setupOptaplannerNativePrJob() {
 void setupOptawebEmployeeRosteringPrJob() {
     def jobParams = getDefaultJobParams('optaweb-employee-rostering')
     jobParams.pr.run_only_for_branches = ['master']
-    jobParams.jenkinsfile = OPTAPLANNER_JENKINSFILE_PATH
+    jobParams.jenkinsfile = BUILD_CHAIN_JENKINS_PATH
     KogitoJobTemplate.createPRJob(this, jobParams)
 }
 
 void setupOptawebVehicleRoutingPrJob() {
     def jobParams = getDefaultJobParams('optaweb-vehicle-routing')
     jobParams.pr.run_only_for_branches = ['master']
-    jobParams.jenkinsfile = OPTAPLANNER_JENKINSFILE_PATH
+    jobParams.jenkinsfile = BUILD_CHAIN_JENKINS_PATH
     KogitoJobTemplate.createPRJob(this, jobParams)
 }
```

Comment:
@rsynek if you are ok with that, I would move those into each `optaweb` repositories to be responsible of their own jobs to create. 
wdyt ?
This would be the same process as explained for quickstarts there: https://github.com/kiegroup/optaplanner-quickstarts/pull/141#issuecomment-883943243